src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -34,6 +36,17 @@
 
 logger = structlog.get_logger(__file__)
 
+pp_curr_version = Version(version("pandapower"))
+pp_ref_version = Version("3.1.2")
```

Comment:
should be all-caps cfr. the PEP standard, and prefixed `_` cfr. our own conventions about private vs public fields

```suggestion
_PP_CURRENT_VERSION = Version(version("pandapower"))
_PP_REF_VERSION = Version("3.1.2")
```